index.js

```diff
@@ -482,6 +482,120 @@ app.post('/api/heartbeat', Validator.body(['coreid', 'event', 'data']).exists(),
   }
 })
 
+app.post('/api/siren-addressed', Validator.body(['coreid', 'event', 'data']).exists(), async (request, response) => {
```

Comment:
Will this webhook be called if someone pushes the "off" button when the siren isn't actually making noise?

Reply:
No, the interrupt is detached after the siren ends

Reply:
So if a person pushes the "off" button a whole bunch of times in quickly in a row, then the first push would stop the interrupt and only one webhook call would be made to this endpoint?

Reply:
Yes - if I'm wrong it would be a hardware thing. I can test this specifically once end to end testing is set up. I've made a note of it.